ceph/rados/core_workflows.py

```diff
@@ -1301,8 +1301,60 @@ def set_mclock_profile(self, profile, osd="osd"):
         Args:
             profile: mclock profile name
             osd: "osd" service by default or "osd.<Id>"
+            reset: revert mClock profile to default - balanced
         """
-        return self.node.shell([f"ceph config set {osd} osd_mclock_profile {profile}"])
+        if self.check_osd_op_queue(qos="mclock"):
```

Comment:
Can we add a check here to see if we are in 6.1 and above, while we test this?

Reply:
Added a check to pass the execution if RHCS build is below 6